src/main/java/io/swagger/swaggerhub/plugin/SwaggerHubUpload.java

```diff
@@ -53,15 +54,17 @@ public void execute() throws MojoExecutionException {
 
         try {
             String content = new String(Files.readAllBytes(Paths.get(inputFile)), Charset.forName("UTF-8"));
+            String oasVersion = DefinitionParserService.getOASVersion( DefinitionFileFormat.valueOf(format.toUpperCase()).getMapper().readTree(content));
```

Comment:
@KevinNewman-SB  I'm mindful of the user parsing an unknown file format which would break the _enum.valueOf(format.toUpperCase())_ ... As is, we don't validate the users configuration. Therefor I think that validation lives outside the context of this PR.

Reply:
Fair enough